### DISCUSSION TITLE
Roadmap needs updating

### DISCUSSION THREAD

**Discussion by imdavidmin** [microsoft/fluentui#20615] [Q&A] [answered]
Looks like the roadmap needs an update - it's currently still talking about Q3 deliveries. No insights on the current progress / roadmap for Q4 and beyond.

https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-Roadmap

**Comment by imdavidmin** [comment, marked as answer]
@JustSlone

**Reply by JustSlone** [reply]
Thanks for the ping. Updated https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-Roadmap
